### PR TITLE
builder/tencentcloud: Correct the image name length validation

### PR DESCRIPTION
**Pull request by shengyu**
Correct the image name length validation, the max length of the TencentCloud image name is 60.

### CODE REVIEW

**Comment by azr** [comment]
Hello @shengyu thanks for opening; the [intl docs](https://intl.cloud.tencent.com/document/product/213/9415?lang=en#2.-input-parameters) says 20. 

@stupidjohn or @likexian, is this okay ? I'm guessing this simply is a discrepancy between the intl docs and the code ?

**Comment by likexian** [comment]
Not OK.
- The limits of tencentcloud image name is `60`, the docs is wrong, I will correct it.
- Since tencentcloud image name is storing as utf8, using `len()` for calculating and limiting to 60 is not suitable, @shengyu would you please change it to `utf8.RuneCountInString()` ?

**Comment by shengyu** [comment]
@likexian @azr Thanks for the review and comments, the commit is updated.

**Comment by shengyu** [comment]
@likexian @azr Thanks for the review and comments, the commit is updated.

File: builder/tencentcloud/cvm/image_config.go [hunk 1]
```diff
@@ -40,11 +41,11 @@ func (cf *TencentCloudImageConfig) Prepare(ctx *interpolate.Context) []error {
 	cf.ForcePoweroff = true
 	if cf.ImageName == "" {
 		errs = append(errs, fmt.Errorf("image_name must be specified"))
-	} else if len(cf.ImageName) > 20 {
-		errs = append(errs, fmt.Errorf("image_name length should not exceed 20 characters"))
+	} else if utf8.RuneCountInString(cf.ImageName) > 60 {
```
Comment (azr) [inline builder/tencentcloud/cvm/image_config.go]: super nice ! 🙂

**Pull request merged**